moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -19,6 +20,32 @@ export const formatProjectActivity = (change, lookupList) => {
     };
   }
 
+  const newRecord = changeData.new;
+  const oldRecord = changeData.old;
+  let changes = [];
+  const fieldsToSkip = [
+    "updated_at",
+    "updated_by_user_id",
+    "project_name_full",
+  ];
+
+  // loop through fields to check for differences, push label onto changes Array
+  Object.keys(newRecord).forEach((field) => {
+    // typeof(null) === "object", check that field is not null before checking if object
+    if (!!newRecord[field] && typeof newRecord[field] === "object") {
+      if (!isEqual(newRecord[field], oldRecord[field])) {
+        changes.push(entryMap.fields[field]?.label);
+      }
+    } else if (
+      newRecord[field] !== oldRecord[field] &&
+      !fieldsToSkip.includes(field)
+    ) {
+      changes.push(entryMap.fields[field]?.label);
+    }
+  });
+
+  const updatedFullName = changes.length > 1;
```

Comment:
Can't remember what `changes` contains here, but is it possible to check that the two changes are actually `project_name` and `project_name_secondary`?

Reply:
@chiaberry I just saw your comments that answer my two questions. 🙏

Reply:
psychic powers!

someday i want to come in and refactor all of the activity log, because I think it could be done better....